test_analyze_job_data.py

```diff
@@ -0,0 +1,11 @@
+from analyze_job_data import analyze_job_data
+import pandas as pd
+
+def test_analyze_job_data():
+    df = pd.read_csv("scraped_data3.csv")
+    analyze_job_data(df)
+    assert True
+
+if __name__ == "__main__":
```

Comment:
Why not just run the tests with pytest, do you still find use for the `main` loop here?

Reply:
I have removed the main loops and implemented proper test formatting for testing the paragraph formatting function, drop duplicates function, and job scraping function.